research-intro.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: page
+title: Introduction to Bitcoin Research
+order: 4
+---
+
+Hello, Researcher!
+
+We appreciate your interest in Bitcoin. 
+
+Bitcoin is a fascinating and multi-faceted research topic.
+It offers challenging questions across a variety of fields of study.
+This page aims to provide an introduction to Bitcoin-related research directions.
+We emphasize computer science (cryptography, privacy, networking) and economics, although Bitcoin can be viewed through the lens of other scientific disciplines too.
+
+Bitcoin offers opportunities for applied as well as fundamental research.
+The former typically designs and evaluates proposed protocol upgrades planned for the next few years.
+The latter tackles more long-term and fundamental challenges that Bitcoin faces.
+
+Bitcoin development emphasizes security and stability.
+Changes to the protocol pass strict review.
+Even though moving academic results towards implementation may be a challenging task,
+high-quality scientific contributions to Bitcoin will have a lasting impact.
+
+We hope this page helps you choose your research direction.
+
+The Bitcoin community looks forward to your contributions!
+
+
+# Introduction
+
+Let us start with introductory materials to get a general overview of the area.
+
+Books:
+
+* Narayanan et al. [Bitcoin and Cryptocurrency Technologies](https://bitcoinbook.cs.princeton.edu/)
+* Antonopoulos. [Mastering Bitcoin](https://aantonop.com/books/mastering-bitcoin/)
+* Antonopoulos et al. [Mastering the Lightning Network](https://github.com/lnbook/lnbook)
+* Rosenbaum. [Grokking Bitcoin](https://www.manning.com/books/grokking-bitcoin)
```

Comment:
The list of books could be very long. What is the criteria for inclusion?

Reply:
It's largely subjective. In particular, the Princeton one is the first academic introductory book on the subject, as far as I know, and I like the fact that it's accompanied by a MOOC.

What changes would you suggest?